__manifest__.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name' : 'Purchase Order Enhancement',
+    'version' : '1.0',
+    'summary': 'Advanced features of Purchase Order',
+    'sequence': 10,
+    'description': """
+        ???
+    """,
+    'website': 'https://www.odoo.com/app',
+    'images' : [],
+    'depends' : ['purchase'],
+    'data': [
+        'security/ir.model.access.csv',
+        'data/purchase_order_enhancement_data.xml',
+        'wizard/archive_old_order_view.xml',
+        'report/purchase_order_enhancement_report.xml',
+        'views/purchase_order_enhancement_views.xml',
+    ],
+    'demo': [],
+    'assets': {
+        'web.assets_backend': [
+            'purchase_order_enhancement/static/src/js/phone_number_widget.js',
+        ],
+    },
+    'installable': True,
+    'application': True,

```

Comment:
This module contains customizations for the purchase module of Odoo. This module itself is not an application. So please mark `'application' `as `False` here